.github/ISSUE_TEMPLATE/3_feature_enhancement.yaml

```diff
@@ -1,10 +1,20 @@
-# docs https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms
-# https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema
-
 name: Propose a feature enhancement
-description: Propose a new feature or an enhancement for Prefect
-labels: [requires triage]
+description: Propose a new feature or an enhancement to an existing feature.
+labels: ["status:needs-triage", "enhancement"]
```

Comment:
I assume this automatically adds labels to the issue? That's awesome!

Reply:
Yeah!